scripts/bdairo.js

```diff
@@ -1,8 +1,11 @@
 let name= "Boluwarin Dairo";
-let id = "INTERNSHIP ID: HNG-01402";
+let id = "HNG-01402";
 let language="Javascript";
-let email = 'abisoyeoladayo@gmail.com';
-let output= " Hello World, This is " + name + " with " + id +" using " +language +" for stage 2 task";
+let email = 'dairobolu@gmail.com';
+let output= "Hello World, this is " + name + " with " + id +" and " + email +" using " + language +" for stage 2 task";
```

Comment:
It is still failing. If you look at [the goal example](https://github.com/hng-teamfury/team-fury-1/blob/master/README.md#the-goal) carefully, you will notice where your output differs.

So please fix accordingly.